docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1275,6 +1282,13 @@ As the core components use emotion as a styled engine, the props used by emotion
 
 [This codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#variant-prop) will automatically update your code.
 
+- Remove the `labelWidth` prop from `InputProps` because it's not longer need.
```

Comment:
I think that we can remove this section. The TextField handle the notions internally. What would be more relevant is to describe the change for OutlinedInput.